vignettes/examples/letterfoot.R

```diff
@@ -0,0 +1,13 @@
+# remotes::install_github("coolbutuseless/minipdf")
+library(minipdf)
```

Comment:
Can we do without additional depends?  I.e. place a pre-made pdf somewhere -- ie in `vignettes/examples` and don't create it on the fly.

Reply:
Oh, and I see you added the file, so this is "mostly" just documentation?

Reply:
Yeah, this just shows how the footer was created.

Reply:
We can do without it if you want.

Reply:
If `R CMD check` does not complain about it (and I later realized it would not as this not package code per se) then it can stay, and is in fact a nice touch documenting how you made it.  

I was just being careful and there is (in my book) no upside to expanding Depends: or Imports: for pure illustrations.

Reply:
OK, thanks. Sure. I myself definitely subscribe to limiting dependencies as much as possible too.